native/explorer/src/dataframe.rs

```diff
@@ -601,7 +601,7 @@ pub fn df_pivot_wider(
         &df,
         values_column,
         id_columns.clone(),
-        [pivot_column],
```

Comment:
I started this Polars bump already 😅 

I had some of the tests fixed. But there is one thing that I couldn't fix, which is the ability to have "ID" columns and new columns repeated (with suffixes). I think I'm going to remove the feature, since Polars is giving errors now.

Reply:
I will close this pr.

Reply:
OK, thanks! I'm going to open a new one in a few minutes. Sorry for not let you know this in the issues :/

Reply:
I did not inform either. No problem, I am ok with closing pr.